hack/clean/clean-libvirt-pool.sh

```diff
@@ -10,10 +10,13 @@ resourcesUsed=""
 getUsedResources() {
     for vm in $(kcli list vm -o json | jq -r .[].name); do
         # preserve ISO/iso used by existing vm's
-        resourcesUsed=$(printf "%s|%s|${resourcesUsed}" "${vm}.ISO" "${vm}.iso")
+        resourcesUsed=$(printf "%s|${resourcesUsed}" "${vm}.ISO")
+
+        # some vm's have a boot-ID iso - depending on how they were created
+        resourcesUsed=$(printf "%s|${resourcesUsed}" "$(kcli show vm ${vm} | grep iso: | awk '{print $2}' | sed 's#\/var\/lib\/libvirt\/images\/##g')")
```

Comment:
if you change the delimiter you don't need to escape it?

Please don't merge, and wait until other colleagues could review it

Reply:
Another colleague already reviewed the code. Also escaping the characters while changing the delimiter does not harm the code.

Reply:
exactly, it does not harm the code, but it's a style and quality of code matter. If we are able to improve something, please do it. (if you don't have enough time I could do it, but let's change it). Also, regarding the tests, please add if you have manual tests to the PR description in the test section, just to ensure everything merged in main has been tested (at least in any way)

Reply:
I disagree, the main reason for code styling is not about code "being pretty" which is highly subjective and not a valid engineering approach, but actually serving a purpose of legibility. Also. it's worth mentioning that styling is *never* enforced, not even in the kernel [1]. Regarding quality, it does have the same quality with or without escaped characters.

In this case the escaped characters cover the, far but possible, problem that a potentially buggy sed release wouldn't escape the characters, and the delimiters are different just for legibility reasons. Anyone familiarized with sed should be capable of reading the command without major issues.

[1] https://www.kernel.org/doc/html/v4.10/process/coding-style.html